azure-pipelines.yml

```diff
@@ -64,11 +68,10 @@ jobs:
             $ANDROID_HOME/platform-tools/adb logcat > $(Build.ArtifactStagingDirectory)/device-${{ value }}.log &
             echo $! > logcat.pid
             echo "Logcat Started with Process ID: $(cat logcat.pid)"
-            $ANDROID_HOME/platform-tools/adb shell "mkdir -p $(DEVICE_VIDEO_DIR)"
             $ANDROID_HOME/platform-tools/adb shell '''
               for i in $(seq 1 1 20)
               do
-                screenrecord --bugreport --verbose --time-limit=180 /mnt/sdcard/DCIM/test"${i}".mp4
+                screenrecord --bugreport --verbose --time-limit=180 $(DEVICE_VIDEO_DIR)/test"${i}".mp4
```

Comment:
Why do we need this change?

Reply:
I changed directory so it will work on all API including 30

Reply:
Should I add a comment?

Reply:
Yes